app/src/js/components/AsyncCommands/AsyncCommands.js

```diff
@@ -138,7 +138,7 @@ AsyncCommand.propTypes = {
   disabled: PropTypes.bool,
   element: PropTypes.string,
   confirmAction: PropTypes.bool,
-  confirmText: PropTypes.string,
+  confirmText: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
```

Comment:
This it to fix the warning message in JavaScript console:
react_devtools_backend.js:4049 Warning: Failed prop type: Invalid prop `confirmText` of type `array` supplied to `AsyncCommand`, expected `string`.
    in AsyncCommand (created by DropdownAsync)
    in DropdownAsync (created by Granule)
    in section (created by Granule)
    in div (created by Granule)
    in Granule (created by Connect(Granule))
    in Connect(Granule) (created by Context.Consumer)
    in withRouter(Connect(Granule)) (created by Context.Consumer)